tools/docker/Dockerfile-inclavare-dev-ubuntu18.04

```diff
@@ -35,6 +35,42 @@ ENV GO111MODULE  on
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --no-modify-path
 ENV PATH         /root/.cargo/bin:$PATH
 
+
+FROM builder AS eaa
+
+# download and install containerd 1.5.3
+RUN wget https://github.com/containerd/containerd/releases/download/v1.5.3/cri-containerd-cni-1.5.3-linux-amd64.tar.gz && \
+    tar zxvf cri-containerd-cni-1.5.3-linux-amd64.tar.gz && \
+    cp -rf etc/* /etc/ && cp -rf opt/* /opt/ && \
+    cp -f usr/local/bin/* /usr/local/bin/ && \
+    rm -rf cri-containerd-cni-1.5.3-linux-amd64.tar.gz etc opt usr
+
+# configure containerd
+RUN containerd config default > config.toml && \
+    sed -i "s#k8s.gcr.io/pause:3.1#registry.cn-hangzhou.aliyuncs.com/acs/pause-amd64:3.1#g" config.toml && \
+    sed -i 's/\(snapshotter = \)"overlayfs"/\1"native"/' config.toml
+
+# build skopeo
+RUN git clone https://github.com/containers/skopeo $GOPATH/src/github.com/containers/skopeo && \
+    apt-get install -y libdevmapper-dev && apt-get install -y build-essential nghttp2 libnghttp2-dev libssl-dev && apt-get install -y libfftw3-dev libsndfile1-dev libgpgme-dev && \
```

Comment:
Please change to `apt-get install -y libdevmapper-dev  build-essential nghttp2 libnghttp2-dev libssl-dev libfftw3-dev libsndfile1-dev libgpgme-dev`